src/buttons/ButtonGroup.js

```diff
@@ -89,12 +89,12 @@ const ButtonGroup = props => {
               },
               i === buttons.length - 1 && {
                 ...lastBorderStyle,
-                borderTopRightRadius: containerBorderRadius,
-                borderBottomRightRadius: containerBorderRadius,
+                borderBottomRightRadius: 3,
```

Comment:
Maybe we can have this in a variable, instead of just having 3's everywhere

Reply:
Thanks! I've set `defaultBorderRadius` variable as 3 👌